packages/xy-chart/src/utils/propShapes.js

```diff
@@ -72,6 +72,22 @@ export const pointSeriesDataShape = PropTypes.arrayOf(PropTypes.shape({
   strokeDasharray: PropTypes.string,
 }));
 
+
+export const PointComponentPropTypes = {
```

Comment:
for consistency I'd just call this `propTypes`

Reply:
I guess you mean `PointComponentDataShape` ?

Reply:
or should I remove it from propShape.js?

Reply:
ah yeah apologies, misread the file. because this isn't a `PropTypes.shape` I would put this in the `PointSeries` component file and export it as `export const pointComponentPropTypes`, then you can import it in `GlyphDotComponent` and others can import it. (also update `index.js`)